### PR TITLE
V2 app

### PR DESCRIPTION
**Pull request by finalspy**
Corrects latest docker image build errors .

- do not update to latest conda 

- force ipython reinstall

- force ConfigSpace dependency lib install with fixed version

- add some comments on those tweaks

- update to latest available saagie/python images

Upgrade of Jupyter version is not part of this build yet

### CODE REVIEW

**Comment by tquiviger** [comment]
ok for me 👍

File: Dockerfile [hunk 1]
```diff
@@ -41,21 +41,35 @@ USER $NB_USER
 RUN jupyter kernelspec remove -f python3
 
 # Update conda to latest version
-RUN conda update -n root conda \
-    && conda clean -afy
+#RUN conda update -n root conda \
```
Comment (PierreLeresteux) [inline Dockerfile]: why comment ? remove will be better

**Review by PierreLeresteux — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (finalspy) [reply Dockerfile]: you're right

Comment (PierreLeresteux) [reply Dockerfile]: *always

:trollface:

**Review by PierreLeresteux — APPROVED** [review]

**Pull request merged**